go/sqlescape/ids.go

```diff
@@ -53,20 +54,29 @@ func EscapeIDs(identifiers []string) []string {
 }
 
 // UnescapeID reverses any backticking in the input string by EscapeID.
-func UnescapeID(in string) string {
+func UnescapeID(in string) (string, error) {
 	l := len(in)
 	if l >= 2 && in[0] == '`' && in[l-1] == '`' {
 		in = in[1 : l-1]
+
+		if !strings.ContainsRune(in, '`') {
```

Comment:
You can directly use `IndexByte` here since that's what ends up being used anyway for single byte characters.

Reply:
Made the change